packages/pn-pa-webapp/src/redux/statistics/actions.ts

```diff
@@ -44,6 +47,7 @@ export const getStatistics = createAsyncThunk(
             lastDate: response.data.lastDate,
             startDate: response.data.startDate,
             endDate: response.data.endDate,
+            isEmpty: isResponseEmpty(response.data as StatisticsResponse),
```

Comment:
Avoid to add an extra field isEmpty. Try to use the redux selector technique https://redux.js.org/usage/deriving-data-selectors

Reply:
ok